invitations/models.py

```diff
@@ -9,7 +9,7 @@
 from django.db import models
 from django.utils import timezone
 from django.utils.crypto import get_random_string
-from django.utils.encoding import python_2_unicode_compatible
+from six import python_2_unicode_compatible
```

Comment:
No point in supporting this now so it can go

Reply:
Removed